Kernel/Bus/USB/USBConfiguration.h

```diff
@@ -33,6 +33,8 @@ class USBConfiguration {
     u8 attributes() const { return m_descriptor.attributes_bitmap; }
     u16 max_power_ma() const { return m_descriptor.max_power_in_ma * 2u; } // Note: "Power" is used incorrectly here, however it's what it's called in the descriptor/documentation
 
+    Vector<USBInterface> const& interfaces() const { return m_interfaces; }
+
     ErrorOr<void> get_interfaces();
```

Comment:
Nitpicky but it may be a good idea to rename this function to something like discover/enumerate_interfaces to avoid confusion about it being a getter.

Reply:
This isn't exactly necessary because Serenity C++ dictates setters are to be bare words, thus there shouldn't really be any confusion :^)

```
Precede setters with the word "set". Use bare words for getters. 
Setter and getter names should match the names of the variables being set/gotten.
```
```
Precede getters that return values through out arguments with the word "get".
```

https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md